pyfar/signals/files.py

```diff
@@ -0,0 +1,526 @@
+    University of Applied Sciences, Anechoic Recordings  [#]_.
+
+    .. note ::
+
+        **License**: CC BY-SA Michio Woirgard, Philipp Stade, Jeffrey Amankwor,
+        Benjamin Bernschütz and Johannes Arend, Audio Group, Cologne University
+        of Applied Sciences
+
+    Parameters
+    ----------
+    sampling_rate : int, optional
+        The sampling rate of the sample in Hz. The default of ``48000`` uses
+        the sample as it is, any other value uses
+        :py:func:`~pyfar.dsp.resample` for resampling to the desired
+        sampling rate.
+
+    Returns
+    -------
+    guitar : Signal
+        The guitar sample.
+
+    References
+    ----------
+    .. [#] http://audiogroup.web.th-koeln.de/anechoic.html
+    """
+
+    # download files if requires
+    files = _load_files('guitar')
+
+    # load brir
+    guitar = pf.io.read_audio(os.path.join(file_dir, files[0]))
+    guitar.fft_norm = "rms"
+
+    # level to make sure all contents have approximately the same loudness
+    guitar.time *= .6
+
+    # resample brir
+    if sampling_rate != 48000:
+        guitar = pf.dsp.resample(guitar, sampling_rate, post_filter=True)
+
+    return guitar
+
+
+def speech(voice="female", sampling_rate=44100):
+    """
+    Get an anechoic speech sample.
+
+    The samples were taken from 'Music for Archimedes' [#]_ (Tracks 4, 5) with
+    kind permission of Bang & Olufsen for research and personal purposes. Any
+    commercial use or publication requires approval of Bang & Olufsen. For more
+    information on the recordings see [#]_.
+
+    .. note ::
+
+        **Copyright**: Bang & Olufsen
+
+    Parameters
+    ----------
+    voice : str, optional
+        Choose between a ``'female'`` (default) and ``'male'`` voice.
+    sampling_rate : int, optional
+        The sampling rate of the sample in Hz. The default of ``44100`` uses
+        the sample as it is, any other value uses
+        :py:func:`~pyfar.dsp.resample` for resampling to the desired
+        sampling rate.
+
+    Returns
+    -------
+    speech : Signal
+        The speech sample.
+
+    References
+    ----------
+    .. [#] Music for Archimedes. Bang & Olufsen, 1992, CD B&O 101.
+
+    .. [#] V. Hansen, and G. Munch, 'Making Recordings for Simulation Tests in
+           the Archimedes Project,' J. Audio Eng. Soc. 39, 768–774 (1991).
+    """
+
+    # download files if requires
+    files = _load_files('speech')
+
+    # load brir
+    file = files[0] if voice == "female" else files[1]
+    speech = pf.io.read_audio(os.path.join(file_dir, file))
+    speech.fft_norm = "rms"
+
+    # level to make sure all contents have approximately the same loudness
+    gain = 0.5 if voice == "female" else 0.3
+    speech.time *= gain
+
+    # resample brir
+    if sampling_rate != 44100:
+        speech = pf.dsp.resample(speech, sampling_rate, post_filter=True)
+
+    return speech
+
+
+def brir(diffuse_field_compensation=False, sampling_rate=48000):
```

Comment:
I'd vote in favor of not using abbreviations as function names. The dark ages of DOS restrictions on string lengths are over :)
```suggestion
def binaural_room_impulse_response(diffuse_field_compensation=False, sampling_rate=48000):
```

Reply:
done